gallery/plot_visualization_utils.py

```diff
@@ -0,0 +1,128 @@
+# image. We can set the colors, labels, width as well as font and font size !
+# The boxes are in ``(xmin, ymin, xmax, ymax)`` format
+# from torchvision.utils import draw_bounding_boxes
+
+from torchvision.utils import draw_bounding_boxes
+
+
+boxes = torch.tensor([[100, 400, 500, 740], [500, 200, 800, 580]], dtype=torch.float)
+labels = ["grass", "racoon"]
+colors = ["blue", "yellow"]
+result = draw_bounding_boxes(racoon_int, boxes, labels=labels, colors=colors, width=10)
+show(result)
+
+#####################################
+# Naturally, we can also plot bounding boxes produced by torchvision detection
+# models.  Here is demo with a Faster R-CNN model loaded from
+# :func:`~torchvision.models.detection.fasterrcnn_resnet50_fpn`
+# model. You can also try using a RetinaNet with
+# :func:`~torchvision.models.detection.retinanet_resnet50_fpn`.
+
+from torchvision.models.detection import fasterrcnn_resnet50_fpn
+
+
+model = fasterrcnn_resnet50_fpn(pretrained=True, progress=False)
+model = model.eval()
+
+outputs = model(racoon.unsqueeze(0))
+print(outputs)
+
+#####################################
+# Let's plot the top 5 boxes detected by our model
+
+boxes = outputs[0]['boxes']
+colors = ["blue", "red", "green", "yellow", "orange"]
+
+result = draw_bounding_boxes(racoon_int, boxes=boxes[:5], colors=colors, width=10)
+show(result)
+
+#####################################
+# Visualizing Segmentation Masks
+# ------------------------------
+# The :func:`~torchvision.utils.draw_segmentation_masks` function can be used to
+# draw segmentation amasks on images. We can set the colors as well as
+# transparency of masks.
+
+from torchvision.utils import draw_segmentation_masks
+from PIL import Image
+from pathlib import Path
+
+
+people = Image.open(Path('assets') / 'people.jpg')
+people = T.ToTensor()(people)
+show(people)
+
+#####################################
+# Let's draw a few maks! The masks contain tensors denoting probabilites of each
+# class.  Here is demo with torchvision's FCN Resnet-50, loaded with
+# :func:`~torchvision.models.segmentation.fcn_resnet50`.
+# You can also try using
+# DeepLabv3 (:func:`~torchvision.models.segmentation.deeplabv3_resnet50`)
+# or lraspp mobilenet models
+# (:func:`~torchvision.models.segmentation.lraspp_mobilenet_v3_large`).
+#
+# Like :func:`~torchvision.utils.draw_bounding_boxes`,
+# :func:`~torchvision.utils.draw_segmentation_masks` requires a single RGB image
+# of dtype `uint8`.
+
+from torchvision.models.segmentation import fcn_resnet50
+
+
+model = fcn_resnet50(pretrained=True, progress=False)
+model = model.eval()
+output = model(people.unsqueeze(0))
+masks = output['out'].squeeze(0)
+
+people_int = T.ConvertImageDtype(dtype=torch.uint8)(people)
+result = draw_segmentation_masks(people_int, masks, alpha=0.2)
+show(result)
+
+#####################################
+# We can adjust alpha to show the masks with a different transparency level:
```

Comment:
I think another thing which would be good to illustrate in the example is how to get the output of Mask R-CNN to be plotted with `draw_segmentation_masks`. People have asked it in the past and I think would be a good thing to showcase as well, as the output format is slightly different

Reply:
I'm getting poor results with maskrcnn_resnet50_fpn, is this expected?
![image](https://user-images.githubusercontent.com/1190450/117685631-01f21400-b1ae-11eb-9094-5b552a0470a2.png)

Reply:
Sure, no worries, thanks for all your work and contributions @oke-aditya !